conversation_classification/kaggle/model.py

```diff
@@ -46,11 +52,17 @@ def estimator_spec_for_softmax_classification(logits, labels, mode):
   """
   predicted_classes = tf.argmax(logits, axis=1)
   predicted_probs = tf.nn.softmax(logits, name='softmax_tensor')
+
   predictions = {
+    # Holds the raw logit values
+    'logits': logits,
+
+    # Holds the class id (0,1) representing the model's prediction of the most
+    # likely species for this example.
     'classes': predicted_classes,
 
-    # Add softmax_tensor to the graph. It is used for PREDICT.
-    'probs': predicted_probs
+    # Holds the three probabilities for each prediction
```

Comment:
Why are there three probabilities?

Reply:
No idea...I think that's a copy/paste typo. I'll fix it.